lib/checks/aria/aria-valid-attr-value-evaluate.js

```diff
@@ -106,30 +107,35 @@ function ariaValidAttrValueEvaluate(node, options, virtualNode) {
     } catch (e) {
       needsReview = `${attrName}="${attrValue}"`;
       messageKey = 'idrefs';
+      return;
```

Comment:
Avoid getting into that if statement below, as we don't want both `needsReview` and `invalid` set.